examples/tutorials/tags.markdown

```diff
@@ -48,7 +48,7 @@ bundle agent cfe_autorun_inventory_listening_ports
 
 In the Enterprise Mission Portal, you can then make a report for
 "Ports listening" across all your machines. For more details, see
-https://cfengine.com/docs/master/guide-enterprise-cfengine-guide-mission-portal-reporting.html
+[Enterprise Reporting](../../enterprise-cfengine-guide/reporting.markdown)
```

Comment:
Here instead of providing an explicit path, we should be able to use the page title as the target.

[Enterprise Reporting][Enterprise Reporting].

The text between the first brackets are the anchor text, and the text between the second brackets is the page title as specified in the page front matter of the page the link should target.

https://raw.githubusercontent.com/cfengine/documentation/master/enterprise-cfengine-guide/reporting.markdown